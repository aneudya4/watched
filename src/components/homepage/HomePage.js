import React from 'react';
import Hero from '../hero/Hero';
import About from '../about/About';
import Features from '../features/Features';
import Login from '../login/Login';
import Register from '../register/Register';

const HomePage = () => {
  return (
    <>
      <Hero />
      <About />
      <Features />
      <Login />
      <Register />
    </>
  );
};
export default HomePage;
