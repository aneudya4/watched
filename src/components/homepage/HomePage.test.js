/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import HomePage from './HomePage';

describe('Test on the <HomePage/> Component', () => {
  it('renders App without crashing', () => {
    const wrapper = shallow(<HomePage />);

    expect(wrapper).toMatchSnapshot();
  });
});
