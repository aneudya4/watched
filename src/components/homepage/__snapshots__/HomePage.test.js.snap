// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on the <HomePage/> Component renders App without crashing 1`] = `
<Fragment>
  <Hero />
  <About />
  <Features />
  <withRouter(Login) />
  <withRouter(Register) />
</Fragment>
`;
